templates/digital-asset-template/frontend/pages/Mint/components/TokenTriImages.tsx

```diff
@@ -0,0 +1,39 @@
+import { Token } from "@/graphql/indexer";
+import { useMemo } from "react";
+import { Image } from "@/components/ui/image";
+
+export function TokenTriImages({
+  tokens,
+  className,
+}: {
+  tokens: Token[];
+  className?: string;
+}) {
+  const threeTokens = useMemo(() => {
+    return tokens.slice(0, 3);
+  }, [tokens]);
+
+  return (
+    <div className={className + " whitespace-nowrap"}>
+      {threeTokens.map((token, i) => (
+        <Image
+          style={{
+            width: `${50 - i * 10}%`,
+            aspectRatio: "1/1",
+            display: "inline-block",
+            position: "relative",
+            left: `-${i * 15}%`,
+            zIndex: -i,
+          }}
+          rounded
+          src={token.cdn_asset_uris?.cdn_image_uri}
+          key={
+            i + " " + token.cdn_asset_uris?.cdn_image_uri ??
+            token.token_name ??
+            i
+          }
+        />
+      ))}
+    </div>
```

Comment:
@blakezimmerman  Positioning these three is interesting. My first inclination was using absolute, but there are odd interactions with proportional sizes and flex containers. 


<img width="289" alt="image" src="https://github.com/aptos-labs/create-aptos-dapp/assets/694324/c32aa5cf-0f84-490e-8eb3-e60d0a6da27f">